### ISSUE TITLE
Connect DailyIntake endpoints

### ISSUE THREAD

**Issue by KyoshiNoda** [KyoshiNoda/Elevate#84]
### Description
- Once #88 is merged into main, please connect the endpoints to the frontend. 
- Create a rtk query file for the given endpoints. 
- Process error handling for it.

### Acceptance Criteria
- New nutritions are added for a given date.
